src/stylesheets/components/_meta.scss

```diff
@@ -1,4 +1,5 @@
 .meta {
+  @include list;
   border-top: solid 1px $grey-3;
   border-bottom: solid 1px $grey-3;
```

Comment:
you could include the border-bottom mixin

Reply:
Correct